pkg/daemon/pivot/utils/run.go

```diff
@@ -48,9 +48,9 @@ func runExtBackoff(backoff wait.Backoff, command string, args ...string) (string
 		return true, nil
 	}); err != nil {
 		if err == wait.ErrWaitTimeout {
-			return "", errors.Wrapf(lastErr, "failed to run command %s (%d tries): %v", command, backoff.Steps, err)
+			return "", fmt.Errorf("failed to run command %s (%d tries): %v: %w", command, backoff.Steps, err, lastErr)
 		}
-		return "", errors.Wrap(err, "failed to run command %s (%d tries): %v")
+		return "", fmt.Errorf("failed to run command %s (%d tries): %v: %w", err)
```

Comment:
This is where the unit test failure is occurring. Basically, this `fmt.Errorf` requires four formatting arguments, but only one was passed in.

This should also fix the lint failure as well.